manifests/j/Jellyfin/Server/10.8.10/Jellyfin.Server.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created with Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Jellyfin.Server
+PackageVersion: 10.8.10
+PackageLocale: en-US
+Publisher: Jellyfin

```

```suggestion
Publisher: The Jellyfin Project
```
to match Control Panel.